packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -303,10 +285,8 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       filled: {
         ...getFilledColorProps(
           {
-            backgroundColor: t.colors.white,
-            color: globalTokens.color.grey[14],
-            backgroundColorDark: globalTokens.color.grey[16],
-            colorDark: t.colors.neutralForegroundOnBrand,
+            backgroundColor: t.colors.neutralBackground1,
+            color: t.colors.neutralForeground1,
             hcBackground: t.colors.neutralBackground3,
             hcColor: t.colors.brandForeground1,
             hcBorderColor: t.colors.brandForeground1,
```

Comment:
>hcBorderColor [](http://example.com/codeflow?start=12&length=13)

Same comment here for border, it can be transparentStroke and then you don't need to specify HC colors.